docs/reference/search/profile.asciidoc

```diff
@@ -916,6 +916,13 @@ The meaning of the stats are as follows:
 
     This is not currently used and will always report `0`. Currently aggregation profiling only times the shard level parts of the aggregation execution. Timing of the reduce phase will be added later.
 
+`build_leaf_collector`::
+    Time spent by the `getLeafCollector()` method collecting documents from
+    segments.
+
+`post_collection`::
+    Time spent by the `postCollect()` method after document collection.
```

Comment:
It looks like this is only used by the cardinality agg at the moment. I purposefully was a little vague here, but we can be more specific if wanted. Let me know your thoughts @nik9000.

Reply:
This has been removed in 7.11, I believe. These aggregations debugging things have a fairly limited audience and I'm not sure how useful it is to document them super thoroughly. We're adding more and more of these specifically so we can ask for them when folks are having trouble and we can see deep into the guts of the system.

For what it is worth the post_collection phase happened after all calls to `collect` were done and before calling the `build_aggregation` phase. We've moved all of the timings for that into the `build_aggregation` phase because of some funny juggling we had to do around delayed collection.